src/transformers/models/bit/modeling_bit.py

```diff
@@ -58,24 +58,34 @@
 
 BIT_PRETRAINED_MODEL_ARCHIVE_LIST = [
     "google/resnetnv2-50",
-    # See all BiT models at https://huggingface.co/models?filter=bit
+    # See all BiT models at https://huggingface.co/models?filter=resnetv2
```

Comment:
To be updated to bit

Reply:
FIxed in [26e8439](https://github.com/NielsRogge/transformers/pull/48/commits/26e843959a427006e9e1ec1080ce2a8a402015aa)